wasmi_v1/src/engine/func_builder/control_stack.rs

```diff
@@ -1,3 +1,5 @@
+use alloc::vec::Vec;
+
 use super::ControlFrame;
```

Comment:
Would you be so kind to merge all the `use` statements in all the changed files? This way `rustfmt` can automatically merge and sort them but not if there is whitespace in between.

Reply:
Done based on your suggestions, with a final cleanup `cargo fmt` on top.

The old sorting was based on some Python habits which I deem rustworthy: imports grouped into paragraphs of standard libraries dependencies and internals.